joomla/joomla-cms
# Added an option to switch username and password reset off

With this PR it is possible to switch the reset username and password option off in com_users and mod_login.

How to test:
Go to Users -> Manage and click the Options button on the top right op the screen

![image1](https://cloud.githubusercontent.com/assets/5610413/9238164/961b0ce2-4150-11e5-8d89-cf96d1f8b50c.jpg)

Turn the 'Allow User Registration' and 'Enable reset options' on and off. Check if the user registration, reset username an reset password checkboxes are turned on and off.

![image2](https://cloud.githubusercontent.com/assets/5610413/9238300/95a77f74-4151-11e5-9779-cb72de0dc6dc.jpg)

Because hiding the links doesn't disable the working of links, they request for both links could be sent anyway. To check if the URL's don't work anymore, first enable the "Enable reset options" switch and click the reset username and password link. Copy both links to a text document. Disable the  "Enable reset options" and paste both links in the browser. There should be an 404 error indication that a reset is not possible.
